Assets/Scripts/UI/ObjectiveList/ObjectiveList.cs

```diff
@@ -9,17 +9,28 @@ public class ObjectiveList : Singleton<ObjectiveList>
     private Image background;
     private TMP_Text objectiveText;
     private Animator root;
+    private bool slideOut;
 
     // Start is called before the first frame update
     void Start()
     {
         background = GetComponentInChildren<Image>();
         objectiveText = GetComponentInChildren<TMP_Text>();
         root =  GetComponentInChildren<Animator>();
-        
         HideObjectiveList();
     }
 
+    void Update()
+    {
+        if (Input.GetKeyDown(KeyCode.Space) && !root.GetBool("slideOut"))

```

Comment:
Can we add "slideOut" into Constants.cs?
Can we also add this to the Input Manager instead of having the control in here? (once you do, make sure to add the name in the Input Manager as a string in Constants.cs as well!)